platform/linuxbsd/export/export.cpp

```diff
@@ -40,8 +40,8 @@ void register_linuxbsd_exporter_types() {
 void register_linuxbsd_exporter() {
 	Ref<EditorExportPlatformLinuxBSD> platform;
 	platform.instantiate();
-	platform->set_name("Linux/X11");
-	platform->set_os_name("Linux");
+	platform->set_name("Linux/BSD (X11)");
```

Comment:
We may as well just ditch the "X11" part of the name now, unless @akien-mga is against it.

Reply:
See [the comment](https://github.com/godotengine/godot/issues/76990#issuecomment-1545380615). I probably misread Akien's comment, and we can remove the "X11" part.

Reply:
Done.